.circleci/config.yml

```diff
@@ -7,13 +7,16 @@ workflows:
   build:
     jobs:
       - elixir/build_test:
+          cache-version: 4
```

Comment:
I bumped it and mix.lock several time, because credo cached some stuff and crashed

Reply:
Yeah, it's a known issue, I believe it has something to do with a recent update of membrane docker image